src/actions/WindowActions.js

```diff
@@ -336,7 +336,12 @@ export function getTab(tabId, windowType, docId) {
         dispatch(getData('window', windowType, docId, tabId))
             .then(res => {
                 if(res.data){
-                    return res.data;
+                    let tab = [];
```

Comment:
It can be better, the tab variable is redundant:
```(...).then(res => res.data && res.data.map(row => Object.assign({}, row, {fields: parseToDisplay(row.fields)}))```